### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/config/frontmatter/portfolio.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#895] [Announcements]
# https://theme-hope.vuejs.press/zh/config/frontmatter/portfolio.html

portfolio 必须设置为 true 以使用档案页面。 home 如果档案是主页时推荐设置为 true name 类型: string 默认值: themeConfig.author.name 档案名称，默认为 themeConfig.author.name 的值。 avatar 类型: string 必填: 否 档案头像图片地址，不支持相对路径。...

https://theme-hope.vuejs.press/zh/config/frontmatter/portfolio.html

**Comment by aquausora** [comment]
medias 配置里的 `link` 似乎是不对的，改成 `url` 才能正常跳转链接
